scripts/gen_csv.py

```diff
@@ -0,0 +1,197 @@
+import os
+from xml.etree import ElementTree
+import pandas as pd
+
+class Patient:
+    def __init__(self):
+        
+        self.text = None
+        self.id = 0
+        
+        self.Asthma = None
+        self.CAD = None
+        self.CHF = None
+        self.Depression = None
+        self.Diabetes = None
+        self.Gallstones = None
+        self.GERD = None
+        self.Gout = None
+        self.Hypercholesterolemia = None
+        self.Hypertension = None
+        self.Hypertriglyceridemia = None
+        self.OA = None
+        self.Obesity = None
+        self.OSA = None
+        self.PVD = None
+        self.Venous = None
+
+
+dic = {} 
+
+# read the first training data
+tree = ElementTree.parse('./text1.xml')
+root = tree.getroot()
+docs  = root.findall('docs')[0]
+for doc in docs:
+    text_data = doc.findall('text')[0].text  # extract texxt data
+    pat = Patient() # create pateint
+    pat.text = text_data
+    pat.id = int(doc.attrib['id'])
+    dic[pat.id] = pat
+
+# read the second training data
+tree = ElementTree.parse('./text2.xml')
+root = tree.getroot()
+docs  = root.findall('docs')[0]
+for doc in docs:
+    text_data = doc.findall('text')[0].text  # extract texxt data
+    pat = Patient() # create pateint
+    pat.text = text_data
+    pat.id = int(doc.attrib['id'])
+    dic[pat.id] = pat
```

Comment:
This is essentially just a repetition of the above, so let's make this a loop instead